release_foreman_rpms

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+if [ -z "$1" ];then
+  echo "Pass version to release, e.g. ./release_foreman_rpms 2.2"
+  exit 1
+fi
+
+VERSION=$1 ./download_rpms
+VERSION=$1 ./sign_rpms
+VERSION=$1 ./upload_rpm_signatures
+VERSION=$1 ./upload_rpms
+VERSION=$1 ./process_rpms
+
+VERSION=$1 PROJECT=client ./download_rpms
+VERSION=$1 PROJECT=client ./sign_rpms
+VERSION=$1 PROJECT=client ./upload_rpm_signatures
+VERSION=$1 PROJECT=client ./upload_rpms
+VERSION=$1 PROJECT=client ./process_rpms
```

Comment:
I'd prefer not to repeat this. That way you can use the same script for all projects (foreman, client, katello). It could even call itself:
```suggestion
if [[ $PROJECT == foreman ]] ; then
    PROJECT=client $0 $1
fi
```

Reply:
Done... with a comment. I prefer being explicit but given this was your original project I've opted for DRY with a comment.